lib/checker.js

```diff
@@ -219,19 +223,35 @@ function sliceImagesOutOfHTMLStringsAndCreateBuffers(readFilesArray) {
 				bufferedImagesOriginal = base64ImagesOriginal.map(
 					function (current, index) {
 						let buffer = Buffer.from(current, 'base64');
-						debugSlice("Created Buffer for Reproduced image #%s: %s", index, buffer.length);
+						debugExtract("Created Buffer for Reproduced image #%s: %s", index, buffer.length);
 						return buffer;
 					}
 				);
 				bufferedImagesReproduced = base64ImagesReproduced.map(
 					function (current, index) {
 						let buffer = Buffer.from(current, 'base64');
-						debugSlice("Created Buffer for Reproduced image #%s: %s", index, buffer.length);
+						debugExtract("Created Buffer for Reproduced image #%s: %s", index, buffer.length);
 						return buffer;
 					}
 				);
 
-				resolve([bufferedImagesOriginal, bufferedImagesReproduced]);
+				/*
+				Sort buffer in an array which contains in each position a further array with the originalImageBuffer and the reproducedImageBuffer.
+				If there are more reproduced buffers than original buffers, they are left out.
+				*/
+				bufferedImagesOriginal.forEach(
+					function (current, index) {
+						resultingImageBuffers[index] = [current, bufferedImagesReproduced[index]]
+					}
+				)
+
+				/*
+				If there are more reproduced images than original images there is a reject.
+				*/
+				if (bufferedImagesOriginal.length < bufferedImagesReproduced.length) {
```

Comment:
Can it only be the case the `original < reproduced` ? Why not test for `!=`?

Reply:
good idea, changed it!